### PR TITLE
Chapter 7 Intro pages

### LINKED ISSUE #827: Chapter 7 Intro Pages
Populate the Intro pages for Chapter 7

image

https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=7103%3A70710&mode=dev

### PR DESCRIPTION
**Pull request by benalleng**
Closes #827 

[Preview](https://saving-satoshi-git-fork-benalleng-issue827-savingsatoshi.vercel.app/en/chapters/chapter-7/intro-1?dev=true)

### CODE REVIEW

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -1507,20 +1507,36 @@ const translations = {
...
+      'You and Mike Ramen arrive first via plane, then via dune buggy, and then via rowboat at Vanderpoole’s private island.',
     intro_one: {
-      title: 'Building blocks',
+      title: 'Intro',
+      paragraph_one:
+        'You ask your compatriots whether they are worried about being spotted by Vanderpoole’s army of security drones.',
+      paragraph_two:
+        '—HOLOCAT: “Give me 15 minutes, and I’ll take care of them. They’re birds, and I’m a cat. They don’t stand a chance.”',
+      paragraph_three:
+        'Holocat extends his claws and deals with Vanderpoole’s drone army. Mike Ramen hands you a black turtleneck sweater, gloves, and a pair of night vision goggles.',
```
Comment (0tuedon) [inline i18n/locales/en.ts:1518]: missing. a "-"  from the design

File: i18n/locales/en.ts [hunk 2]
```diff
@@ -1507,20 +1507,36 @@ const translations = {
...
+      paragraph_four:
+        '—MIKE RAMEN: “You can’t do espionage in a Hawaiian shirt. What were you thinking? This isn’t another abandoned warehouse; this is a fortress. Here, I almost forgot. Take this grappling hook.”',
+      paragraph_five:
+        'You and Mike Ramen scale the castle walls of Vanderpoole’s primary residence. To your surprise, his home overflows with pro-bitcoin art, books, and paper magazines. He is or rather was, a true bitcoiner. A collection of miners dating back 125 years sit anointed in display cases. That’s when you see it: Vanderpoole’s workshop.',
```
Comment (0tuedon) [inline i18n/locales/en.ts:1522]: I see, that's intentional? because it's not part of the conversation

File: i18n/locales/en.ts (hunk 1 above)
Comment (benalleng) [reply i18n/locales/en.ts:1518]: ahh yeah, I was referencing the updated designs with the latest images and the "-" was removed

File: i18n/locales/en.ts (hunk 2 above)
Comment (benalleng) [reply i18n/locales/en.ts:1522]: yeah I think in the figma the "-" wasn't intentional

**Pull request merged**